theforeman.org/yaml/views/release.yml

```diff
@@ -13,6 +13,7 @@
     minor:
         - '7'
         - '8'
+        - '9'
```

Comment:
This is called out as a manual TODO:
https://github.com/theforeman/jenkins-jobs/blob/b993393b01c86082cefda9cfea5dfbba5abd0d3c/branch-foreman#L31-L32